common.blocks/progressbar/progressbar.bh.js

```diff
@@ -1,10 +1,18 @@
 module.exports = function(bh) {
 
     bh.match('progressbar', function(ctx, json) {
-        var val = json.val;
+        var val = json.val || 0;
 
         ctx
-            .js({ val : val })
+            .js({ val : json.val })
```

Comment:
in this case you will get an error when using getVal(), if BEMJSON hasn't `val`.